third_party/libunwindstack/tests/DwarfDebugFrameTest.cpp

```diff
@@ -726,7 +747,7 @@ TYPED_TEST_P(DwarfDebugFrameTest, GetFdeFromOffset32_lsda_address) {
   SetCie32(&this->memory_, 0x5000, 0xfc,
            std::vector<uint8_t>{/* version */ 1,
                                 /* augment string */ 'z', 'L', '\0',
-                                /* address size */ 8,
+                                // TODO: Note that address size is not supported in version == 1.
```

Comment:
I don't get this comment. Is this really a TODO?

Reply:
Yes, this was left in by mistake. I had marked this to consider what to do here, but I believe the appropriate thing is to just delete this line. There is no address size in version 1, so the data as setup before was simply not correct. 

Done (removed TODO line).